operator-pipeline-images/operatorcert/schemas/ci-schema.json

```diff
@@ -0,0 +1,68 @@
+{
+    "title": "ci.yaml validation schema",
+    "description": "json schema validating the ci.yaml",
+    "type": "object",
+    "additionaProperties": true,
+    "properties": {
+        "fbc": {
+            "description": "Config set for FBC-enabled operator bundles",
+            "type": "object",
+            "properties": {
+                "updateGraph": {
+                    "description": "Name of the update strategy for the operator",
+                    "type": "string"
+                },
+                "reviewers": {
+                    "description": "List of authorized GitHub usernames",
+                    "type": "array",
+                    "items": {
+                        "type": "string",
+                        "minItems": 1
+                    }
+                },
+                "merge": {
+                    "description": "Flag that determines whether the PR is set to auto-merge",
+                    "type": "boolean"
+                },
+                "cert_project_id": {
+                    "description": "Certification Project ID linked with the operator",
+                    "type": "string"
+                },
+                "enabled": {
+                    "description": "Flag that determines whether the operator uses FBC delivery method",
+                    "type": "boolean"
+                },
+                "catalog_mapping": {
+                    "description": "Catalog Mapping accoering to provided templates types",
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "template_name": {
+                                "description": "Name of the catalog template file",
+                                "type": "string"
+                            },
+                            "catalog_names": {
+                                "description": "List of the catalogs",
+                                "type": "array",
+                                "items": {
+                                    "type": "string"
+                                },
+                                "minItems": 1
+                            },
+                            "type": {
+                                "description": "Type of template schema",
+                                "type": "string",
+                                "enum": ["olm.template.basic", "olm.semver"]
+                            }
+                        },
+                        "required": [ "template_name", "type", "catalog_names" ]
+
+                    }
+                }
+            },
+            "minItems": 1,
+            "required": [ "catalog_mapping" ]
```

Comment:
The catalog mapping is not a mandatory field. It depends on a fbc.enabled flag. Please keep it optional in the schema.